cmd/restic/cmd_rewrite.go

```diff
@@ -164,6 +193,14 @@ func filterAndReplaceSnapshot(ctx context.Context, repo restic.Repository, sn *r
 			Verbosef("would remove old snapshot\n")
 		}
 
+		if metadata != nil && metadata.Time != nil {
+			Verbosef("would set time to %s\n", metadata.Time)
+		}
+
+		if metadata != nil && metadata.Hostname != "" {
+			Verbosef("would set time to %s\n", metadata.Hostname)
```

Comment:
I believe this should be "would set hostname"?

Reply:
Indeed. Thanks, the typo will be fixed by https://github.com/restic/restic/pull/4669